src/Servers/Kestrel/Core/src/Internal/ConfigurationReader.cs

```diff
@@ -65,6 +80,10 @@ public IEnumerable<EndpointConfig> Endpoints
             }
         }
 
+        private void ReadClientCertificateMode()
```

Comment:
This should do the enum parsing instead of LoadClientCertificateMode in KestrelConfigurationLoader.

Reply:
Compare to ParseProtocols